controls/radspreadsheet/features/ui-headers-and-footers.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Headers and Footers
+page_title: Headers and Footers
+description: Headers and Footers
+slug: radspreadsheet-ui-headers-and-footers
+tags: header,footer
+published: True
+position: 6
+---
+
+# Headers and Footers
+
+
+Headers and Footers functionality allows you add rich text content in the page margins when exporting a worksheet to pages. This feature is useful in scenarios like [exporting to PDF](http://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/formats-and-conversion/pdf/pdfformatprovider), [printing]({%slug radspreadsheet-ui-printing%}) or [import/export to XLSX](http://docs.telerik.com/devtools/document-processing/libraries/radspreadprocessing/formats-and-conversion/xlsx/xlsxformatprovider). 
```

Comment:
Is exporting the proper word here? Maybe something like "lay out a worksheet to pages" would be better.